libs/payments/stripe/src/lib/stripe.manager.spec.ts

```diff
@@ -26,6 +27,33 @@ describe('StripeManager', () => {
     manager = module.get<StripeManager>(StripeManager);
   });
 
+  const mockPayPalSubscription1 = SubscriptionFactory({
+    collection_method: 'send_invoice',
+    status: 'active',
+  });
+
+  const mockPayPalSubscription2 = SubscriptionFactory({
+    collection_method: 'send_invoice',
+    status: 'past_due',
+  });
+
+  const mockPayPalSubscription3 = SubscriptionFactory({
+    collection_method: 'send_invoice',
+    status: 'trialing',
+  });
+
+  const mockStripeSubscription = SubscriptionFactory({
+    collection_method: 'charge_automatically',
+    status: 'active',
+  });
```

Comment:
Factories should be used as close as possible to your tests, if possible. That way you're only creating the test data you absolutely need in a test, as well as guaranteeing no side-effects if one of your tests (or test subjects) happen to modify the data.